content/funding-your-training.md

```diff
@@ -101,29 +112,33 @@ below.
 
 $bursaries-and-scholarships$
 
-## Tuition fee and maintenance loans
+(CTA TO Chat to us here)
+If you're unsure about whether you qualify for a bursary or scholarship you can chat to us
+Chat online
 
-You can apply for a tuition fee loan to pay for your training so you don't need to pay anything upfront. You can also apply for a maintenance loan to help with living costs and you may be able to get a [bursary or a scholarship](#bursaries-and-scholarships).
+### Non-graduate bursaries
 
-Even if a bursary or scholarship is available for your course, you can get student finance to cover your fees and help with your living costs. You can apply for a tuition fee and/or a maintenance loan even if you already have a student loan.
+If you're a final year undergraduate student and your course started in the academic year 2021 to 2022, you may be eligible for a training bursary of £9,000. The bursary is available if you're studying a QTS course in secondary maths or physics, or you're studying an opt-in QTS course in secondary undergraduate computing, languages, mathematics or physics.
 
-You can apply for a tuition fee loan of up to £9,250 to cover the full cost of your course fees and a maintenance loan of up to £12,382 to help with your living costs.
+There are [Troops to Teachers] (https://www.ucas.com/teaching-option/troops-teachers) tax-free bursaries of £40,000 for former military personel  undertaking  undergraduate degrees leading to Qualified Teacher Status (QTS) in England. You may be eligible for the bursary if you train to teach secondary biology, physics, chemistry, computing, maths or modern foreign languages.
```

Comment:
There's a space here between the link text and target, it should be `[Troops to Teachers](https://something)`

```suggestion
There are [Troops to Teachers](https://www.ucas.com/teaching-option/troops-teachers) tax-free bursaries of £40,000 for former military personel  undertaking  undergraduate degrees leading to Qualified Teacher Status (QTS) in England. You may be eligible for the bursary if you train to teach secondary biology, physics, chemistry, computing, maths or modern foreign languages.
```